var/spack/repos/builtin/packages/qt6declarative/package.py

```diff
@@ -3,9 +3,10 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-from spack import *
 import os
 
+from spack import *
```

Comment:
These are apparently superfluous in current versions of spack -- you can delete the line entirely.